gateway/server.go

```diff
@@ -151,6 +151,7 @@ func main() {
 	if faasHandlers.QueuedProxy != nil {
 		r.HandleFunc("/async-function/{name:[-a-zA-Z_0-9]+}/", faasHandlers.QueuedProxy).Methods(http.MethodPost)
 		r.HandleFunc("/async-function/{name:[-a-zA-Z_0-9]+}", faasHandlers.QueuedProxy).Methods(http.MethodPost)
```

Comment:
I think it'd make more sense just to expand the `Methods()` to include both for this.

My recollection is that you asked me about this recently, but on reflection I'm not sure what the use-case is for an asynchronous `GET`. The queue-worker may have also needed a change. For now I think we should pause this PR.